account_banking_payment_export/wizard/payment_order_create.py

```diff
@@ -83,7 +80,7 @@ def filter_lines(self, lines):
         """
         self.ensure_one()
         payment_lines = self.env['payment.line'].\
-            search([('order_id.state', 'in', ('draft', 'open')),
+            search([('order_id.state', '!=', 'cancel'),
                     ('move_line_id', 'in', lines.ids)])
```

Comment:
If you have used the transfer account, it shouldn't be a problem, but in other cases... both filters can work, but with further modules. I'm working now on the abandoned _account_payment_return_ that allows to reflect returns in your payment orders. This module reconciles the original invoice move line, with the payment order move line and a new generated move line for the return. This way, you get the same balance, an history is created, and you can pass the new move line with another due date to a new payment order. In this flow, the new filter is the good one.

Anyway, @ecino, can you please propose both improvements in separate split so that we can get as soon as possible the computed total (which is something that I need urgently), and discuss the other also with the context of the other module?